lib/msal-browser/src/controllers/StandardController.ts

```diff
@@ -521,26 +521,22 @@ export class StandardController implements IController {
             .then((result: AuthenticationResult | null) => {
                 if (result) {
                     this.browserStorage.resetRequestCache();
+                    this.eventHandler.emitEvent(
+                        EventType.ACQUIRE_TOKEN_SUCCESS,
+                        InteractionType.Redirect,
+                        result
+                    );
+                    this.logger.verbose(
+                        "handleRedirectResponse returned result, acquire token success"
+                    );
                     // Emit login event if number of accounts change
                     const isLoggingIn =
                         loggedInAccounts.length < this.getAllAccounts().length;
                     if (isLoggingIn) {
                         this.eventHandler.emitEvent(
                             EventType.LOGIN_SUCCESS,
```

Comment:
I think we can emit this event in BrowserCacheManager.setAccount to centralize it (where you removed the ACCOUNT_ADDED event)